packages/target-truffle-v5-test/types/truffle-contracts/DataTypesInput.d.ts

```diff
@@ -67,7 +67,7 @@ export interface DataTypesInputInstance extends Truffle.ContractInstance {
     input1: number | BN | string,
     input2: number | BN | string,
     txDetails?: Truffle.TransactionDetails
-  ): Promise<[BN, BN]>;
+  ): Promise<{ 0: BN; 1: BN }>;
```

Comment:
funny that for truffle the change is excatly the opposite of the ethers change :sweat_smile:

Reply:
Yup :D but anyway, now types correspond to real runtime values so it's fine.